tests/integration/renderer/test_due_diligence.py

```diff
@@ -0,0 +1,114 @@
+import json
+import os
+import string
+
+from bs4 import BeautifulSoup
+import requests
+
+import plotly
+import dash_html_components as html
+import dash
+
+
+def test_rddd001_initial_state(dash_duo):
+    app = dash.Dash(__name__)
+    my_class_attrs = {
+        "id": "p.c.4",
+        "className": "my-class",
+        "title": "tooltip",
+        "style": {"color": "red", "fontSize": 30},
+    }
+    # fmt:off
```

Comment:
in general, I'm happy with the unarguable `black` default style, and here is an exception I explicitly turned it off